src/rohan/TripletInArray.java

```diff
@@ -0,0 +1,38 @@
+package rohan;
+
+/*
+ find the triplet having maximum sum.
+[10,12,8,20,18,22,15,13,17]
+output : maximum sum triplet 20,18,22 --> 60
+ */
+
+public class TripletInArray {
+	public static void main(String[] args) {
+		TripletInArray tripletInArray = new TripletInArray();
+		int[] numArr = {10,12,8,20,18,22,15,13,17,14,25,60};
+		tripletInArray.tripletSumCount(numArr);
+	}
+	
+	void tripletSumCount(int[] numArr) {
+		int sum=0;
+		String tmpStr="";
+		int tmpSum=0;
+		String str="";
+		if(numArr.length%3 == 0) {
+			for(int index=0;index<numArr.length;) {
+				sum=0;
```

Comment:
Hi Rohan, we can handle this in single it self
Check this code,
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/priyanka/Assignment28/MaxSumOfTriplet.java